pandas/_libs/tslibs/strptime.pyi

```diff
@@ -1,11 +1,10 @@
-from typing import Optional
-
 import numpy as np
 
 def array_strptime(
     values: np.ndarray,  # np.ndarray[object]
-    fmt: Optional[str],
+    fmt: str | None,
     exact: bool = True,
-    errors: str = "raise"
+    errors: str = "raise",
```

Comment:
while cleaning these stubs can also remove the default values and just use `...`

from https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#conventions

> Stub files should only contain information necessary for the type checker, and leave out unnecessary detail:

> for arguments with a default, use ... instead of the actual default;